packages/host/app/components/operator-mode/stack-item.gts

```diff
@@ -536,7 +536,7 @@ export default class OperatorModeStackItem extends Component<Signature> {
                     @realmInfo={{realmInfo}}
                     @canAnimate={{this.isTopCard}}
                     class='header-icon'
-                    style={{cssVar
+                    @style={{cssVar
```

Comment:
I solved this differently in https://github.com/cardstack/boxel/pull/1700/files#diff-b758e2686ae55e1164a0876f12f04ccf876deea30f42d531e2ff721c239fdc71R46-R49

Reply:
your situation seems different though. In my case I need to set the `style` attr in order to set the background URL. that will collide with any style that is passed in via splattributes. I changed from using an `<img>` to `<div>` with background url because it was not possible to get the light and dark animation effect with a `<img>` alone.